modules/file-system/pre-existing-network-storage/templates/ddn_exascaler_luster_client_install.tftpl

```diff
@@ -46,3 +46,5 @@ do
 done
 chmod +x /usr/sbin/esc-client
```

Comment:
do you think the error here will be clear if the curl never succeeded? (maybe it will)

Reply:
Each curl failure is printed in the startup script, so it will look like 20 curl failures and then a file does not exist. To me that is pretty clear what happened but maybe it could be made better. Do you have a suggestion?

Reply:
`set -e` has been added to this script and tested (as discussed offline).